src/components/Image/DetailV2/ImageContestCollectionDetails.tsx

```diff
@@ -58,6 +59,16 @@ export const ImageContestCollectionDetails = ({
                   Contest{tagDisplay}.
                 </Text>
 
+                {!!item.collection.metadata?.votingPeriodStart && (
+                  <Text>
+                    The ability to react/vote for this film will go live starting at{' '}
+                    <Text weight="bold" component="span">
+                      {formatDate(item.collection.metadata?.votingPeriodStart)}
```

Comment:
I think format date doesn't include time by default, it'd be a good idea to modify the format here

Reply:
That's ok I think, the date we record is a date not a datetime